doc/HOWTO_oracle.md

```diff
@@ -227,3 +230,98 @@ impdp \
   REMAP_SCHEMA="bc_nucore:c##nucore_open_development"
   REMAP_TABLESPACE="bc_nucore:USERS"
 ```
+
+
+## Troubleshooting
+
+### Known issues on Apple Silicon computers
+
+#### Oracle instant client architecture
+
+If you are using an arm64 architecture you may have installed an arm64 ruby version.
+
+When running the bundle install there's the oracle gem called ruby-oci8 that requires you to have Oracle instant client locally. If you followed the [Install Oracle Instant Client](#Install-Oracle-Instant-Client) section on this readme, you have the 86x version of the Oracle instant client locally and this may cause conflicts while installing the gem. Installing the x64 version of the client does the job and allows you to install the gem but it causes issues while running the rails server.
+
+The solution is to install ruby 86x version, unluckily reinstalling ruby with a different architecture would make your other local repositories that use the same ruby version stop working.
+This issue can be solved by having 2 instances of the same ruby version but for different architectures simultaneously.
+
+1. You are going to need Homebrew installed for both architectures so we are going to teach the terminal which Homebrew path to use depending on the current selected architecture.
+
+Add the following code to your `~/.profile`, `~/.zshrc` or `~/.bash_profile` depending on your terminal's platform.
+
+```
+if [ "$(uname -m)" = "arm64" ]; then
+  echo "arm64"
+  export PATH="/opt/homebrew/bin:${PATH}"
+  [ -f /opt/homebrew/bin/brew ] && eval "$(/opt/homebrew/bin/brew shellenv)"
+else
+  echo "$(uname -m)"
+  # Postgres Caveats
+  export PATH="/usr/local/opt/libpq/bin:$PATH"
+  export LDFLAGS="-L/usr/local/opt/libpq/lib"
+  export CPPFLAGS="-I/usr/local/opt/libpq/include"
+  export PKG_CONFIG_PATH="/usr/local/opt/libpq/lib/pkgconfig"
```

Comment:
Why do we need to set this? Is it related to Oracle?

Reply:
You're right; this is not strictly necessary here. These exports help when you are running other projects locally that use Postgres.